miner/worker.go

```diff
@@ -562,7 +562,10 @@ func (w *worker) mainLoop() {
 			if w.isRunning() && w.current != nil && len(w.current.uncles) < 2 {
 				start := time.Now()
 				if err := w.commitUncle(w.current, ev.Block.Header()); err == nil {
-					w.commit(w.current.copy(), nil, true, start)
+					err := w.commit(w.current.copy(), nil, true, start)
+					if err != nil {
+						log.Error("Commit error", "error", err)
+					}
```

Comment:
In general, when you have a situation like this where you only need the `err` very locally, it's a bit cleaner to write it like this: 
```suggestion
					if err := w.commit(w.current.copy(), nil, true, start); err != nil {
						log.Error("Commit error", "error", err)
					}
```
I think that goes for all the cases below. That can also be useful to avoid overwriting an already present variable. So instead of 'reusing` (`err = ...`), you define a new one with the same name for that scope only. That way it's trivial to see that the change has no side-effects.